wireapp/wire-android
# feat: Search and Display Services when Adding Participants (AR-3318)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3318" title="AR-3318" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-3318</a>  Add services to a group conversation as an admin
  </td></table>
  <br />
 






































----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

There was no way to add a new service to a group conversation.

### Causes (Optional)

Not implemented.

### Solutions

Add a new view pager into `Add participants` screen so it now has two tabs (PEOPLE / SERVICES) where the same search input will be in effect for both.

Needs releases with:

- [ ] [#1737 on Kalium](https://github.com/wireapp/kalium/pull/1737)

### Testing

#### How to Test

- Open App
- Open a Group Conversation
- Click on Add Participants
- PEOPLE and SERVICES tab should be shown
- Can search for services while in the Services tab
- Clicking on a service will lead you to the Service Details screen [implemented here](https://github.com/wireapp/wire-android-reloaded/pull/1767)

### Attachments (Optional)
<img src="https://github.com/wireapp/wire-android-reloaded/assets/5890660/ca5ef7b7-b74e-4e58-bdc9-d784943c2a21" width="200" height="400" alt="services list" />
<img src="https://github.com/wireapp/wire-android-reloaded/assets/5890660/1f4bc141-0cd9-495d-8f99-55131f3e24d5" width="200" height="400" alt="services search" />
